src/ngx_http_lua_common.h

```diff
@@ -236,6 +236,17 @@ union ngx_http_lua_srv_conf_u {
 
 
 typedef struct {
+    ngx_int_t                is_handler_inline;
+    u_char                  *rewrite_chunkname;
+    ngx_http_complex_value_t rewrite_src;    /*  rewrite_by_lua
+                                                inline script/script
+                                                file path */
+
+    u_char                  *rewrite_src_key; /* cached key for rewrite_src */
+} ngx_http_lua_rewrites_t;
```

Comment:
Seems like we can reuse this struct for other phases as well? So we do not need to encode "rewrite" into this struct's names.